### DISCUSSION TITLE
Notifications for subscription changes

### DISCUSSION THREAD

**Discussion by joemasilotti** [pay-rails/pay#634] [Q&A]
Is it possible to get notified when a subscription status changes?

From my understanding all changes are processed via a web hook handled by Py. Ideally, that web hook response could call into an object in my own codebase notifying me of the changes.

I'm looking for ways to update a reporting dashboard instead of manually checking statuses every hour or so.

**Comment by excid3** [comment]
Register your own webhook listener. 👍

https://github.com/pay-rails/pay/blob/master/docs/7_webhooks.md#custom-webhook-listeners

**Reply by joemasilotti** [reply]
Oh, perfect! Not sure how I missed this from the docs, thanks Chris.

To sanity check, the internal Pay web hook stuff runs before the custom listener is called. Right?
